src/applications/static-pages/renderHomepageBanner.js

```diff
@@ -0,0 +1,51 @@
+import yaml from 'js-yaml';
+import * as Sentry from '@sentry/browser';
+
+const HOMEPAGE_BANNER_YML_LOCATION =
+  'https://raw.githubusercontent.com/department-of-veterans-affairs/vagov-content/master/fragments/home/banner.yml';
+
+async function loadConfigFromGitHub() {
+  const response = await fetch(HOMEPAGE_BANNER_YML_LOCATION, {
+    mode: 'cors',
+  });
+
+  const homepageBannerYml = await response.text();
+  return yaml.safeLoad(homepageBannerYml);
+}
+
+function renderToDocument(banner) {
+  if (!banner.visible) {
+    return;
+  }
+
+  const root = document.getElementById('homepage-banner');
+
+  root.innerHTML = `
+    <div class="usa-alert-full-width usa-alert-full-width-${banner.type}">
```

Comment:
is it at all worth doing a sanity check on the `type`, `title`, and `content` props we get from the YAML before just throwing it in the DOM?

Reply:
Changes still have to go through code review in vagov-conent. Also added sanitation.